.gitlab-ci.yml

```diff
@@ -0,0 +1,11 @@
+variables:
+  NOMAD_VAR_HOSTNAMES: '["colsearch.crawl1.archive.org"]'
+  NOMAD_VAR_CHECK_PROTOCOL: 'tcp'
+  NOMAD_VAR_FORCE_PULL: 'true'
+  NOMAD_VAR_CPU: 2000
+  NOMAD_VAR_MEMORY: 2048
```

Comment:
Is it possible to move this variables to the web interface of our GitLab instead of including here in the repo?

Reply:
Yes it is